### PR TITLE
Added the pairwise loss function in lossfunc.py addressing issue #32

### PR DESCRIPTION
**Pull request by riakashyap**
I've implemented the loss function. Comment if anything else needs to be added.

image

### CODE REVIEW

**Comment by bdunahu** [comment]
The implementation looks correct to me. Can you explain why it's better to subclass nn.Module? Why a class at all and not just a standalone function?

**Comment by riakashyap** [comment]
Having the loss as a class gives flexibility for future work, especially if we want to introduce temperature scaling or different weighting schemes, as described in trainer.py.

**Review by bdunahu — APPROVED** [review]

**Review by uchuuronin — APPROVED** [review]
lgtm
